### ISSUE TITLE
CSS styling not found

### ISSUE THREAD

**Issue by joaolisboa** [ParticleCore/Particle#631]
Not sure if this is an issue with Github since the CSS files used by both Particle or Iridium seem to be in your Github page particlecode.github but I figured I'd write it here.

Since Friday, 95+% of the time the browser fails to reach the address of the url in both Particle and Iridium and causes the youtube layout to be broken, the top bar covers part of the videos and obviously everything related to the extension stops working.

Would it be possible to include the CSS inside the javascript file to fix this issue? As I've said I don't know the origin of the issue, the only thing I know I changed was the DNS in my computer to Google's DNS but I don't see why that would cause the issue. The CSS url is still reached sometimes but rarely.

**Comment by ParticleCore** [comment]
I have not found any issues with the resource distribution, make sure you follow the troubleshooting steps to rule out anything that might be interfering from your side: https://github.com/ParticleCore/Particle/wiki/Report-a-problem#troubleshooting

As for including in the files, that is not going to happen since it would be rejected from AMO store.

**Comment by joaolisboa** [comment]
Unfortunately all of that falls into my troubleshooting routine which I've already tried with no result. This is a recent thing only which is weird. I've used this script for more than a year without issue which is unfortunate now that this issue occurs.

It's really weird, because I just can't seem to get into this url, with/without version: https://particlecore.github.io/Particle/stylesheets/YouTubePlus.css

It just can't be reached. I've tried going into Iridium but same thing happened so I might as well stick with YT+ since it has more features.
